Gemfile

```diff
@@ -5,6 +5,8 @@ ruby '3.0.3'
 
 gem 'rubocop', '>= 1.0', '< 2.0'
```

Comment:
Project looks great but it's missing a professional README. A professional README file should have a project title, description, and instructions to set the project up locally.

Reply:
Thank you, I forgot to set it up.